version.txt

```diff
@@ -1 +1 @@
-5.1
+5.2.1
```

Comment:
Deliberate move to 3-part versioning?

Reply:
I've botched 5.2 on test.pypi.org, decided to add another part. 😅

Reply:
If it's just for test.pypi.org, could just force overwrite? Feels a bit weird to change versioning scheme because of a little test-upload versioning problem. (If you're happier with 3-part, though, go for it!)

Reply:
Switched back to 2-part